onnxruntime/core/session/onnxruntime_c_api.cc

```diff
@@ -93,39 +67,121 @@ class LoggingWrapper : public ISink {
   void* logger_param_;
 };
 
+struct OrtEnv {
+ public:
```

Comment:
nit: maybe 'class' is more appropriate given what is in it now

Reply:
Yup, will take care of in another PR.